src/common/asb/Asb.c

```diff
@@ -1634,8 +1630,14 @@ static char* AuditEnsureLoggingIsConfigured(void* log)
 static char* AuditEnsureSyslogPackageIsInstalled(void* log)
 {
     char* reason = NULL;
-    CheckPackageInstalled(g_syslog, &reason, log);
-    CheckPackageInstalled(g_rsyslog, &reason, log);
+    if ((0 == CheckPackageInstalled(g_systemd, &reason, log)) && (0 == CheckPackageInstalled(g_rsyslog, &reason, log)))
+    {
+        return reason;
+    }
+    if ((0 == CheckPackageInstalled(g_systemd, &reason, log)) && (0 == CheckPackageInstalled(g_syslog, &reason, log)))
```

Comment:
Is it intentional to check `(0 == CheckPackageInstalled(g_systemd, &reason, log)` here and also above?

Reply:
This return is only checked in the OR cases, aka skip rest when one of them succeeds. All other do not need the result codes, as reasons texts grow with each call and besides the reasons themselves, indicate pass or fail.